### DISCUSSION TITLE
Custom Font

### DISCUSSION THREAD

**Discussion by ghost** [btcpayserver/btcpayserver#2280] [Technical Support] [answered]
As @artdesignbysf tells me, currently BTCPay does not allow to upload a custom font. It does allow to set a custom header & CSS though.

A commonly used way to get a custom font is to import a google [:vomiting_face: :rage:] font via CSS. For obvious reasons, this is not an option for me.

It would be great to allow a custom font upload, so that no external server has to be pinged for getting the font.

**Comment by artdesignbySF** [comment, marked as answer]
I might have been mistaken @MaxHillebrand. Blame it on too much screen time, too little bedtime ;)
Can I upload any font to the file server in BTCPay and link the css to that? I don't see why not... Or am I missing something?

Sorry if this is a moot issue because of my probable blunder!

**Reply by ghost** [reply]
fwiw, I couldn't find anything about custom font upload in the docs neither... only the google css import.

**Reply by dennisreimann** [reply]
@artdesignbySF yes, just upload and use it, this should work.

**Reply by ghost** [reply]
that's great, thanks for clarifying @dennisreimann!!

**Reply by artdesignbySF** [reply]
Good to know. Thanks @dennisreimann for the assist.

**Reply by dennisreimann** [reply]
You're welcome – let me know in case something doesn't work :)
